sdk/src/wallet/core/operations/stronghold_backup/mod.rs

```diff
@@ -103,7 +105,10 @@ impl Wallet {
             .password(stronghold_password.clone())
             .build(backup_path.clone())?;
 
-        let (read_client_options, read_coin_type, read_secret_manager, read_accounts, chrysalis_data) =
+        #[cfg(feature = "storage")]
```

Comment:
Why does this need the storage feature? Stronghold alone should be sufficient, no?

Reply:
No, see the feature gate below

Reply:
You added the feature there, but the function also doesn't need it?

Reply:
both have it?

Reply:
Why does `migrate_snapshot_from_chrysalis_to_stardust()` need it?

Reply:
Because otherwise it's unused

Reply:
But why not have the stronghold feature instead of storage?

Reply:
because we use storage below this...

Reply:
You mean for `.set(CHRYSALIS_STORAGE_KEY, &chrysalis_data)`? `migrate_snapshot_from_chrysalis_to_stardust()` does more than only returning this data

Reply:
I see, so we should call it regardless of whether we will use the result?

Reply:
I would say so

Reply:
Hopefully it's fixed now